src/python/ensembl/io/genomio/gff3/process_gff3.py

```diff
@@ -507,10 +509,17 @@ def transfer_description(self, gene: SeqFeature) -> None:
                     # No transcript product, but a CDS product? Copy it to both transcript and gene
                     for cds in tran.sub_features:
                         if cds.type == "CDS" and "product" in cds.qualifiers:
+                            print(f"Transfer description from CDS to gene and transcript in {gene.id}")
                             description = cds.qualifiers["product"][0]
                             tran.qualifiers["product"] = [description]
                             gene.qualifiers["product"] = [description]
+                        else:
+                            print(f"No transfer possible for {gene.id}")
                     # Continue transfering the translation products to the transcripts
+                else:
+                    print(f"Transfer not allowed: {gene.id} <- {tran.id} ({tran.type})")
+        else:
+            print("No transfer")
```

Comment:
```suggestion
            print("No product description")
```
just a suggestion as it was more clear to me

Reply:
removed, that method is no longer in use (moved to functional_annotation.pm)